api/order/order.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+const orderSchema = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  email: String,
+  status: String,
+});
```

Comment:
Should we incorporate tickets array so they are accessible here already? [tickets.id]